solved_exercises/marius/rsp.py

```diff
@@ -0,0 +1,65 @@
+"""
+Rock Scissor Paper is a well known and fun game
+Just sellecrt R, S or P and have fun
+"""
+
+import sys
+import random
+
+"""<< [r]ock [p]aper [s]cissor or quit << p
+
+ >> round   3
+ >> [player]     rock   vs   paper    [computer] <<
+ ..:: computer wins ::..  reason: paper covers rock.
+ >>   1 (50.00 %)  --  (50.00 %)   1"""
+
+
+def play_rock_scissors_paper():
```

Comment:
Please separate the concerns as we discussed in our live session:
* reading inputs
* evaluate the winner
* couting stats
* printing results